ctk/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -92,6 +93,7 @@ class TestCommon {
 
         val idpMetadata = Common.parseIdpMetadata()
         val SP_ISSUER = spMetadata.keys.first()
+        val SP_ENTITY_INFO = checkNotNull(spMetadata[SP_ISSUER])
 
         val acsUrl: Map<SamlProtocol.Binding, String?> by lazy {
             val spInfo = spMetadata[SP_ISSUER]
```

Comment:
✏️ We can use `SP_ENTITY_INFO` here instead of doing ` val spInfo = spMetadata[SP_ISSUER]`